examples/sound_capture/SoundCapture.cpp

```diff
@@ -3,6 +3,7 @@
 // Headers
 ////////////////////////////////////////////////////////////
 #include <SFML/Audio.hpp>
+#include <cstdlib>
```

Comment:
I would move this include under `<iostream>`.

Reply:
Fixed.